openlibrary/solr/solr_types.py

```diff
@@ -1,5 +1,6 @@
 # This file is auto-generated by types_generator.py
 # fmt: off
+# ruff: noqa: UP045
```

Comment:
Note: we can remove this in a future PR by changing the script that generates these to use the `| None` syntax instead of Optional.